README.md

```diff
@@ -15,6 +15,8 @@ Over 1500 Free SVG icons for popular brands. See them all on one page at <a href
 
 ## Usage
 
+> ℹ We ask that all users read our [legal disclaimer](./DISCLAIMER.md) before using icons from Simple Icons.
```

Comment:
I'm _pretty_ sure this will render correctly on NPM and Packagist as well.

```suggestion
> :information_source: We ask that all users read our [legal disclaimer](./DISCLAIMER.md) before using icons from Simple Icons.
```